openedx/features/course_experience/__init__.py

```diff
@@ -33,11 +29,12 @@ def course_home_page_title(course):  # pylint: disable=unused-argument
     return _('Course')
 
 
-def default_course_url_name(request=None):
+def default_course_url_name():
     """
     Returns the default course URL name for the current user.
     """
-    if waffle.flag_is_active(request or RequestCache.get_current_request(), UNIFIED_COURSE_VIEW_FLAG):
+    # TEMPORARY NOTE: Does this break because we no longer pass through the request?
+    if COURSE_OUTLINE_PAGE_FLAG.is_enabled():
```

Comment:
@andy-armstrong The course id wasn't always available to send to this function.  So the options are either:
1. Go with a WaffleFlag (rather than a CourseWaffleFlag) that doesn't allow for course override, or
2. Find a way to get the course (id) on to the request object as we discussed a long time ago, or
3. Some other idea to get this to work if you have anything.

For now, to see what tests pass/fail with this change, I went with option 1.  The major problem with this option is that all community instances of Open edX will need to roll-out the same way we did, which is to say, with a big bang roll-out with no option to delay rollout for certain courses.